vendor/github.com/prometheus/client_golang/prometheus/testutil/testutil.go

```diff
@@ -42,9 +42,8 @@ import (
 	"fmt"
 	"io"
 	"net/http"
-	"reflect"
 
-	"github.com/davecgh/go-spew/spew"
+	"github.com/kylelemons/godebug/diff"
```

Comment:
looks like this was responsible for pulling in https://github.com/prometheus/client_golang/pull/1539 and github.com/kylelemons/godebug